### ISSUE TITLE
Any plans to put out a nuget package?

### ISSUE THREAD

**Issue by SamuelCox** [wolfgarbe/PruningRadixTrie#10]
Hi,
Thanks for this, I have a custom implementation of Tries that this could likely replace. Are there any plans to upload this as a nuget package? I appreciate it's only 2 classes of code which can easily be copied, but was just wondering.

**Comment by wolfgarbe** [comment]
Yes, but I'm not sure when I will find time.

**Comment by SamuelCox** [comment]
Fair enough! Thanks. If you want help creating a pipeline for it then I'm happy to.
